.NET/Microsoft.Recognizers.Definitions.Common/Italian/NumbersWithUnitDefinitions.cs

```diff
@@ -484,8 +486,8 @@ public static class NumbersWithUnitDefinitions
 			{ @"Centimetro cubo", @"cm3|cm^3|cm³|centimetro cubo|centimetri cubi|centimetro cubico|centimetri cubici" },
 			{ @"Millimetro cubo", @"mm3|mm^3|mm³|millimetro cubo|millimetri cubi|millimetro cubico|millimetri cubici" },
 			{ @"Ettolitro", @"ettolitro|ettolitri" },
-			{ @"Decalitro", @"dal|decalitro|decalitri|dekalitro|dekalitri" },
-			{ @"Litro", @"l|litro|litri" },
+			{ @"Decalitro", @"decalitro|decalitri|dekalitro|dekalitri" },
+			{ @"Litro", @"litro|litri" },

```

Comment:
'l' 'dal' and 'le' were removed because they are confused with Italian determiners and prepositions. 
adding them to ambiguousunit was not effective.

Reply:
"dal" is indeed a not so frequently used unit and I'd be Ok with it not being recognized for now.
Can you add here two sentence examples with "dal" in Italian? One that mentions the real unit and one that showcases a false positive match?

Reply:
However, removing "l" is not acceptable as is. "l" is a very common unit used in daily life.
Can you also provide similar examples of correct mentions and false positives, so we can look at alternative workarounds?

Reply:
If this is the only pending issue, I suggest **opening another PR for Currency/Temperature/Age** (so those can be merged quickly) and keep **this one just for Dimension pending this discussion**.

Reply:
@tellarin we will create a separate PR.

Reply:
@LionbridgeCS, please also reply to these:
- "dal" is indeed a not so frequently used unit and I'd be Ok with it not being recognized for now.
Can you add here two sentence examples with "dal" in Italian? One that mentions the real unit and one that showcases a false positive match?
- However, removing "l" is not acceptable as is. "l" is a very common unit used in daily life.
Can you also provide similar examples of correct mentions and false positives, so we can look at alternative workarounds?

Reply:
"la tanica contiene 120 dal di etanolo" (dal unit)
"dal 1996 abbiamo ridotto i consumi del 23%" (dal preposition)
***********************************************************************************************
"abbiamo raccolto 200 l d'acqua" (l unit)
"ricordo che l'8 marzo ha fatto molto caldo" (l determiner)